importer/SeShipSv.py

```diff
@@ -118,6 +119,13 @@ def set_monuments_all_id(self):
         """
         self.monuments_all_id = self.signal
 
+    def set_descriptions(self):
+        """Set default descriptions."""
+        DESC_BASES = {"sv": "K-märkt fartyg",
```

Comment:
Done (I guess this assumes that all the items in the db are of this type, but it doesn't look like any of the items in [2] are also included in [1]).